### PR TITLE
capnproto: update to 0.10.2

### PR DESCRIPTION
**Pull request by marmeladema**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

**Comment by marmeladema** [comment]
Alright, ready for review/merge :+1:

File: common/shlibs [hunk 1]
```diff
@@ -2924,6 +2924,17 @@ libkj-http-0.9.1.so capnproto-0.9.1_1
...
+libpng.so.3 libpng12-1.2.56_2
+libcapnpc-0.10.2.so capnproto-0.10.2_1
+libcapnp-json-0.10.2.so capnproto-0.10.2_1
+libcapnp-rpc-0.10.2.so capnproto-0.10.2_1
+libcapnp-websocket-0.10.2.so capnproto-0.10.2_1
+libcapnp-0.10.2.so capnproto-0.10.2_1
+libkj-http-0.10.2.so capnproto-0.10.2_1
+libkj-async-0.10.2.so capnproto-0.10.2_1
+libkj-test-0.10.2.so capnproto-0.10.2_1
+libkj-0.10.2.so capnproto-0.10.2_1
```
Comment (paper42) [inline common/shlibs:2937]: libpng12 is not touched in this template, is this a mistake? Also, 0.9.1.so won't be available anymore, so these lines should be replaced, not added.

Comment (marmeladema) [reply common/shlibs:2937]: Hum sorry, it's a git rebase mistake.

Comment (marmeladema) [reply common/shlibs]: Fixed!

**Comment by marmeladema** [comment]
@paper42 sorry for the delay, it should be good to go now

File: srcpkgs/capnproto/template [hunk 2]
```diff
@@ -10,7 +10,7 @@ maintainer="eater <hello@eaterofco.de>"
 license="MIT"
 homepage="https://capnproto.org"
 distfiles="${homepage}/capnproto-c++-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/capnproto/template:12]: expand $homepage here

**Review by paper42 — COMMENTED** [review]
pre_configure and hostmakedepends can be removed in capnproto

File: srcpkgs/capnproto/template (hunk 2 above)
Comment (marmeladema) [reply srcpkgs/capnproto/template]: Done

**Comment by marmeladema** [comment]
> pre_configure and hostmakedepends can be removed in capnproto

Done

**Pull request merged**